src/globals/global.css

```diff
@@ -0,0 +1,13 @@
+/* This file is for global styling. */
```

Comment:
@alihejazi  Please see the comments in the other 2 `globals` files for why this is probably not a good idea.